### PR TITLE
[IMP] account_multicurrency_revaluation

### PR DESCRIPTION
**Pull request by andrius-preimantas**

### CODE REVIEW

**Comment by andrius-preimantas** [comment]
Backport of 12 changes + a few extra changes to work with Deeper case.

**Comment by andrius-preimantas** [comment]
Backport v12 changes + a few changes to solve corner cases (e.g. invoice in foreign currency, payment in company currency, etc)

File: account_multicurrency_revaluation/readme/USAGE.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ To use this module, you need to:
   *Allow currency revaluation* checked.
 * Open the report through the following menu:
   Accounting > Reporting > Legal Reports > Accounting Reports > Print Currency Unrealized
+<<<<<<< HEAD
```
Comment (naglis) [inline account_multicurrency_revaluation/readme/USAGE.rst:9]: :mag:

**Review by naglis — APPROVED** [review]
Did not review as it is not clear what is backport and what is our changes.

**Review by psugne — APPROVED** [review]

**Pull request merged**